examples/python/access_token_generator.py

```diff
@@ -0,0 +1,53 @@
+import httpx
+import time
+import jwt
+from cryptography.hazmat.backends import default_backend
+from cryptography.hazmat.primitives import serialization
+from typing import Dict
+from data_structures import PrivateApiKey
+
+
+class AccessTokenGenerator:
+
+    @staticmethod
+    def generate(
+        identity_provider_url: str,
+        project_identifier: str,
+        private_api_key: PrivateApiKey,
+    ) -> str:
+        try:
+            private_key = serialization.load_pem_private_key(
+                private_api_key.key.encode(), password=None, backend=default_backend()
+            )
+
+            headers = {"kid": private_api_key.key_id}
+
+            current_time = int(time.time())
+
+            claims = {
+                "iat": current_time,
+                "iss": private_api_key.user_id,
+                "sub": private_api_key.user_id,
+                "aud": identity_provider_url,
+                "exp": current_time + 3600,
```

Comment:
I think this should be like so just to respect our IdP settings:
```suggestion
                "exp": current_time + 1800,
```